test/helpers/setup-migration.js

```diff
@@ -30,21 +30,18 @@ export default function setupMigrationTest(filename, mocks) {
     ...mocks.mockPrompts
   });
 
-  let run = mockCommands({
+  let run = await mockCommands({
     npm: () => ({ status: 0 }),
     bundle: () => ({ status: 0 }),
     ...mocks.mockCommands
   });
 
-  mockRequire('fs', {
-    existsSync: path => path.includes('/.codeshift/') ||
+  spyOn(fs, 'existsSync').and.callFake(path => {
```

Comment:
Let's keep the existing spy intact. 

```suggestion
  fs.existsSync.and.callFake(path => {
```

Probably not worth the extra work, but we could also use matchers and `returnsValue` to achieve what we're doing in the fake without interrupting any mock filesystem stuff.